### DISCUSSION TITLE
Zapret через LAN а не WAN

### DISCUSSION THREAD

**Discussion by Mihrutkin** [bol-van/zapret#388] [Q&A]
господа, подскажите по Zapret.
Если роутер подключен к другому роутеру c интернетом через lan и wan не задействован.
Как в этом случае использовать запрет?

Команда не помогла: OPENWRT_LAN="lan1 lan2 lan3 lan4"

**Comment by bol-van** [comment]
Эта конфигурация называется "мост"
https://github.com/bol-van/zapret/discussions/262
